pennylane/_qubit_device.py

```diff
@@ -78,6 +78,24 @@ def __init__(self, wires=1, shots=1000, analytic=True):
         """None or array[int]: stores the samples generated by the device
         *after* rotation to diagonalize the observables."""
 
+    @classmethod
+    def capabilities(cls):
+        """Get the capabilities of the plugin.
+
+        Devices that inherit from this class automatically
+        have the following items in their capabilities
+        dictionary:
+
+        * ``"model": "qubit"``
+        * ``"tensor_observables": True``
+
+        Returns:
+            dict[str->*]: results
+        """
+        capabilities = cls._capabilities
+        capabilities.update(model="qubit", tensor_observables=True)
+        return capabilities
+
```

Comment:
Just a question here: would there be a reason against having    `` _capabilities = {"model"="qubit", "tensor_observables"=True}`` here and later calling ``dict.update()`` in the plugins? I feel having the base ``dict`` defined here would be more natural, but honestly either case seems to be equally good solutions to me at this point

Reply:
This is a very good point, and I think that way around would be preferable. I only did it this way because I was trying to preserve `_capabilities` being a simple class attribute in the child device (and you can't call `update` on a class attribute).

Is there a way of keeping it a class attribute in the child class, but 'inheriting' some defaults from the parent class?

Reply:
Evidently, there is a way to achieve this behaviour, but it's tricky. See [the second answer here](https://stackoverflow.com/questions/2488306/python-inheritance-of-a-class-attribute-list).

Reply:
Oh nice! I would say though, best to avoid complex logic (especially if it involves metaclasses) for a small change such as this, it could lead to maintenance complexity down the line.

Reply:
Yeah, guess you're right!

Reply:
Yes, this sounds reasonable, thanks for the examples!